api_types.ts

```diff
@@ -761,6 +770,14 @@ export interface Sync {
    * {@link core.MetadataContext} The parameters of the sync formula for the sync table.
    */
   readonly parameters?: MetadataContext;
+
+  /**
+   * This is only populated when `context.invocationLocation.source === InvocationSource.Brain`.
+   *
+   * TODO(patrick): Unhide this
+   * @hidden
+   */
+  syncType?: BrainSyncType;
```

Comment:
Optional, wondering if we should call this `brainSyncType` if we think it will only ever describe things specific to Brain.